custom/icds_reports/templates/icds_reports/icds_app/navigation.directive.html

```diff
@@ -137,6 +137,12 @@
                     <div class="col-sm-9">{% trans "AWC Report" %}</div>
                 </a>
             </li>
+            <li ng-class="{'active': $location.path().indexOf('/lady_supervisor') !== -1}">
```

Comment:
Should not this be under feature flag. It looks like this html code will always show the ls dashboard menu item

Reply:
Fixed, thanks.